introduction.tex

```diff
@@ -1,33 +1,33 @@
 \label{sec:introduction}
 Molecular dynamics (MD) simulations are a powerful method to generate new insights into the function of biomolecules \citep{Borhani:2012mi, Dror:2012cr, Orozco:2014dq, Perilla:2015kx, Bottaro:2018aa}.
 These simulations produce trajectories, timeseries of atomic coordinates, that now routinely include millions of timesteps and can be Terabytes in size.
-These trajectories need to be analyzed using statistical mechanics approaches \cite{Mura:2014kx} but because of the increasing size of the data, analysis of the trajectories is becoming a bottleneck in typical scientific workflows in the biomolecular simulation community \cite{Cheatham:2015}.
-Many data analysis tools and libraries have been developed to extract the desired information from the output trajectories from MD simulations ~\cite{nmoldyn, nmoldyn-2012, Hum96, Hinsen:2000kx, Grant:2006ud, himach-2008, Romo:2009zr, Romo:2014bh, Michaud-Agrawal:2011fu, Gowers:2016aa, cpptraj-2013, mdtraj-2015, pteros2015, Doerr:2016aa} but few can make efficient use of modern high performance computing (HPC) resources to accelerate the analysis stage.
-As a concrete example we focus on the \package{MDAnalysis} package \citep{Gowers:2016aa,Michaud-Agrawal:2011fu}, which is an open-source object-oriented Python library for structural and temporal analysis of molecular dynamics simulation trajectories and individual protein structures.
-Although \package{MDAnalysis} accelerates selected algorithms with Open-MP, full scale parallel trajectory analysis has not yet been implemented in the library.
-Here we discuss the challenges and lessons-learned for making parallel analysis on HPC resources feasible for a general purpose trajectory analysis library.
+These trajectories need to be analyzed using statistical mechanics approaches \cite{Mura:2014kx} but because of the increasing size of data, trajectory analysis is becoming a bottleneck in typical biomolecular simulation scientific workflows~\cite{Cheatham:2015}.
+Many data analysis tools and libraries have been developed to extract the desired information from the output trajectories from MD simulations ~\cite{nmoldyn, nmoldyn-2012, Hum96, Hinsen:2000kx, Grant:2006ud, himach-2008, Romo:2009zr, Romo:2014bh, Michaud-Agrawal:2011fu, Gowers:2016aa, cpptraj-2013, mdtraj-2015, pteros2015, Doerr:2016aa} but few can efficiently use modern High Performance Computing (HPC) resources to accelerate the analysis stage.
+We focus on \package{MDAnalysis}~\citep{Gowers:2016aa,Michaud-Agrawal:2011fu}, which is an open-source object-oriented Python library for structural and temporal analysis of molecular dynamics simulation trajectories and individual protein structures.
+Although \package{MDAnalysis} accelerates selected algorithms with Open-MP, distributed parallel trajectory analysis has not yet been implemented in the library.
+Here we discuss the challenges and lessons-learned for making parallel analysis on HPC resources feasible for MDAnalysis.
```

Comment:
I believe this should remain "a general purpose trajectory analysis library". Because the presents guidelines are applicable to other python packages and are not limited to MDAnalysis. We mentioned this in other sections of the paper as well.

Reply:
Changed it back to
      Here we discuss the challenges and lessons-learned for making parallel analysis on HPC resources feasible in a general purpose trajectory analysis library such as \package{MDAnalysis}.